src/garage/torch/q_functions/continuous_mlp_q_function.py

```diff
@@ -0,0 +1,40 @@
+"""This modules creates a continuous MLP Q fcuntion network."""
+
+
+class ContinuousMLPQFunction:
+    """
+    This class implements a q value network.
```

Comment:
No need for "This class implements" -- you can always start with "Implements a Q-Value network..."